app/scripts/controllers/swaps.js

```diff
@@ -455,22 +485,51 @@ export default class SwapsController {
               ),
               10,
             )
-            .minus(tokenConversionRate ? ethFee.toString(10) : 0, 10)
+            .minus(tokenConversionRate ? totalEthCost : 0, 10)
```

Comment:
Review note: this is functionally equivalent. What used to be called `ethFee` is now `totalEthCost`. (What is now `ethFee` is now different from `totalEthCost` when `sourceToken === ETH_SWAPS_TOKEN_ADDRESS)`